### PR TITLE
[INLONG-9795][Sort] Optimize the definition of enumeration variables and remove the semicolon

### LINKED ISSUE #9795: [Improve][Sort] Regarding the format optimization of the data type enumeration definition of the Redis connector project module.
### Description

The current enumeration class org.apache.inlong.sort.redis.common.config.RedisDataType is defined as,
     ; symbol ends. Although there is no grammatical problem with this, the code looks confusing, so it is recommended to modify it and remove it.
     ; symbol makes the code look clearer.

### InLong Component

InLong Sort

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by YaziLee**
Fix #9795

### Prepare a Pull Request

- Title Example: [INLONG-9795][Sort] Optimize the definition of enumeration variables…

### Motivation
The current enumeration class org.apache.inlong.sort.redis.common.config.RedisDataType is defined as,
     ; symbol ends. Although there is no grammatical problem with this, the code looks confusing, so it is recommended to modify it and remove it.
     ; symbol makes the code look clearer.

### Modifications
Removed the trailing symbol "" of the enumeration class org.apache.inlong.sort.redis.common.config.RedisDataType,
     ;".

### Verifying this change
None

### Documentation

  - Does this pull request introduce a new feature? (yes / no) no
  - If yes, how is the feature documented? (not applicable / docs / JavaDocs / not documented)
  - If a feature is not applicable for documentation, explain why?
  - If a feature is not documented yet in this PR, please create a follow-up issue for adding the documentation

### CODE REVIEW

**Comment by dockerzhang** [comment]
@YaziLee thanks for your contribution, please create an issue to describe the problem.

**Comment by EMsnap** [comment]
please change the title [INLONG-hello] to inlong-issueNumber

File: inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/redis/src/main/java/org/apache/inlong/sort/redis/common/config/RedisDataType.java [hunk 1]
```diff
@@ -47,6 +47,5 @@ public enum RedisDataType {
      * Since strings are binary safe blobs and their maximum length is 512 MB, <br/>
      * they are suitable to set up to 2^32 different bits.
      */
-    BITMAP,
-    ;
+    BITMAP
```
Comment (dockerzhang) [inline inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/redis/src/main/java/org/apache/inlong/sort/redis/common/config/RedisDataType.java:50]: "BITMAP;" ?

Comment (YaziLee) [reply inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/redis/src/main/java/org/apache/inlong/sort/redis/common/config/RedisDataType.java:50]: Yes, thank you for your suggestion.

**Review by aloyszhang — APPROVED** [review]
LGTM

**Review by dockerzhang — APPROVED** [review]

**Pull request merged**